.github/workflows/dependabot-labeler.yaml

```diff
@@ -0,0 +1,17 @@
+# This workflow file adds the 'lgtm' and 'approved' labels to Dependabot PRs
+# This is done to ensure that the PRs that pass e2e are automatically merged/added to merge-queues by the CodeFlare bot
+name: Dependabot Labeler
```

Comment:
Can you use [this](https://github.com/actions/labeler)?

Reply:
Good call, it looks like we can pass the `GITHUB_TOKEN` too. Will update this thanks!

Reply:
I've had a closer look at this, I think it’s a great feature but I believe for this particular use case it won’t work as expected. After some digging into how the labeler action works, I see it can automatically add specific labels based on the files that have been changed. I think this is great, and we should look to implement this for notebook or ui changes for example. 

The issue in this case is that the CodeFlare Machine needs to add the `approved` and `lgtm` labels to add the PR to the merge queue, so it wouldn't be ideal to make it add these labels based on file changes as these could be performed by different actors such as another user.